servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -273,6 +274,15 @@ public void createBlockRemote(long sessionId, long blockId, int tierAlias, long
   }
 
   /**
+   * Gets the block store which contains all blocks in the manager.
+   *
+   * @return the block store
+   */
+  public BlockStore getBlockStore() {
+    return mBlockStore;
+  }
```

Comment:
I think this breaks the abstraction a bit too much, would it be possible to just use the BlockDataManager to access the block store? (Ie. using the currently provided methods in the interface)